frontend-project/src/pages/channels/ChannelsDetailView.tsx

```diff
@@ -0,0 +1,232 @@
+import { PageHeaderWrapper } from '@ant-design/pro-layout';
+import { Button, Col, Card, Form, Input, Row, Space, Spin, Switch } from 'antd';
+import { connect, useDispatch } from 'dva';
+import React, { useEffect, useState } from 'react';
+import { formatMessage, FormattedMessage } from 'umi-plugin-react/locale';
+import { Channel } from '@/services/definitions';
+import { ReduxResourceState } from '@/utils/reduxModel';
+import router from 'umi/router';
+import { localeKeys } from '../../locales/pl-PL';
+import { RouterTypes } from 'umi';
+import { ServiceResponse } from '@/services/service';
+import smallEodSDK from '@/utils/sdk';
+import { openNotificationWithIcon } from '@/models/global';
+
+interface ChannelsDetailViewProps {
+  channels: ReduxResourceState<Channel>;
+  match: RouterTypes['match'] & { params: { id: string | undefined } };
+}
+
+const layout = {
+  labelCol: { span: 8 },
+  wrapperCol: { span: 16 },
+};
+
+const tailLayout = {
+  wrapperCol: { offset: 8, span: 16 },
+};
+
+function ChannelsDetailView({ channels, match }: ChannelsDetailViewProps) {
+  const dispatch = useDispatch();
+  const isEdit = Boolean(match.params.id);
+  const [isSubmitting, setIsSubmitting] = useState(false);
+  const editedChannel = channels.data.find(value => value.id === Number(match.params.id));
+  const [form] = Form.useForm();
+  (window as any).se = smallEodSDK;
```

Comment:
Ajj, widzę że zostawiłem narzędzie do debugowania. Usuniesz? Co do #716 to tam nie musisz. Wpadnie poprawka na Sprawy i Tagi.

Reply:
Rozumiem, że chodzi o linię 35 ? :)

Reply:
Wyciągałem nazwy metod w API, bo nie mamy otypowanego SDK.

Reply:
SDK ma dokumentacje: https://github.com/watchdogpolska/small-eod-sdk-javascript/#documentation-for-api-endpoints